app/packs/src/components/sampleTaskInbox/SampleTaskCard.js

```diff
@@ -172,13 +172,17 @@ const SampleTaskCard = ({ sampleTask }) => {
         {panelHeading()}
         {deleteButton()}
       </Card.Header>
-      <Card.Body className="d-flex">
-        <div className="w-50">
-          {contentForSample(sampleTask, dropRef)}
-        </div>
-        <div className="w-50">
-          {contentForSampleTask(sampleTask)}
-        </div>
+      <Card.Body>
+        <Container>
```

Comment:
You do not have to have a container in order to use row. No extra nesting needed.

Reply:
Interestingly enough though you are right in this case, the Footer of the modal (SampleTaskInbox.js) needs a container around the row, otherwise the Dropzones don't use the full width of the row... no idea why...
So I have two different places where I want to achieve the same thing (2 content blobs that should span a full row next to each other) with different behaviour. That being the way, I would either understand why this happens or have at least consistent markup :D

Reply:
CSS is awesome

Reply:
If you want them side by side, why does the footer have an flex-column class?

Reply:
Some remains from experimenting with the utility classes if I remember correctly.
I updated the corresponding codepaths, so it looks a bit better now :)